script/Arcueid/Day 5/05_00_ARC05_1.txt

```diff
@@ -299,12 +299,13 @@ Not a single victim has been discovered―――?
 [sha:9945f275c190c472f276e6b4aecace1a00ee64ce]{
 -- Page 25, Offset 8650.
 -- なぜそこまで解っていてはっきりと口にしない。
-How can you say that when you have so much information?
+// im not sure this is right, but the newscaster shouldn't know what really happened right?
+How can you say that when we already know?
```

Comment:
How can you say that when it's already so clear what happened?

Everyone that stayed in the hotel was...